mozdownload/scraper.py

```diff
@@ -100,6 +100,7 @@ def __init__(self, destination=None, platform=None,
         logging.basicConfig(format=' %(levelname)s | %(message)s')
         self.logger = logging.getLogger(self.__module__)
         self.logger.setLevel(log_level)
+        logging.getLogger('redo').setLevel(logging.INFO)
```

Comment:
I would not go to do this for requests at the same time though - or maybe we can add it in a separate commit, not squashed at the end ?